src/workspace.ts

```diff
@@ -65,14 +65,44 @@ export class WorkspaceDataProvider implements TreeDataProvider<WorkspaceItem> {
 export class WorkspaceItem extends TreeItem {
 	constructor(
 		label: string,
-		rClass: string[],
+		rClass: string,
+		str: string,
 		type: string,
-		typeDetailed: string,
 		length: number,
 		collapsibleState: TreeItemCollapsibleState
 	) {
 		super(label, collapsibleState);
-		this.description = typeDetailed;
+		this.description = str;
 		this.tooltip = `${label} (${rClass}, length of ${length})`;
 	}
 }
+
+export function saveWorkspace(): void {
+	window.showSaveDialog({
+		defaultUri: Uri.file('workspace.RData'),
```

Comment:
Looks like it will go to the root folder every time the dialog is open. Should we make a Uri from the workspace folder? For example, get the active document uri, and get workspace folder from that uri. If no workspace folder contains the active document, then we go to the folder of the active document. If there's no active folder, then we go to user home?

Reply:
Agreed, I'll work on a fix